src/main/java/usecase/alter_entry/AlterEntryController.java

```diff
@@ -0,0 +1,15 @@
+package usecase.alter_entry;
+
+import billgates.database.QueryEntryData;
+
+public class AlterEntryController {
+    private final AlterEntryInputPort useCase;
+
+    public AlterEntryController(AlterEntryInputPort useCase) {
+        this.useCase = useCase;
+    }
+
+    public void alter_Entry(int billId, int entry_id, Object new_value,String alter_column) {
```

Comment:
Convention: Please use a consistent style. In java, we used lower camelcase as normal variable and method names.
For example, `alter_Entry` should be changed to `alterEntry`.

Reply:
Also, `billId` should not be a parameter. 

We can get the current bill id by calling `User.getInstance().getCurrentBillId()` in the use case. Since the one that used `alterEntry` here is in the drivers layer, it shouldn't have access to `User`.